### PR TITLE
ko: cr2 Bare Metal translations

### PR DESCRIPTION
**Pull request by henrif75**
#262 #1463
*Stats*
Before: 1297 translated messages, 383 fuzzy translations, 1780 untranslated messages.
After: 1497 translated messages, 430 fuzzy translations, 1533 untranslated messages.

### CODE REVIEW

File: po/ko.po [hunk 1]
```diff
@@ -4683,12 +4685,14 @@ msgid "/// The car has arrived on the given floor.\n"
 msgstr ""
 
 #: src/user-defined-types/exercise.md:29 src/user-defined-types/solution.md:44
+#, fuzzy
 msgid "/// The car doors have opened.\n"
-msgstr ""
+msgstr "/// 보내려면 지웁니다.\n"
```
Comment (jiyongp) [inline po/ko.po:4690]: a

**Comment by henrif75** [comment]
Stats after clean-up:
1497 translated messages, 391 fuzzy translations, 1572 untranslated messages.

File: po/ko.po [hunk 2]
```diff
@@ -15632,8 +15636,11 @@ msgid "_Embed.toml_:"
 msgstr "Embed.toml:"
 
 #: src/bare-metal/microcontrollers/debugging.md:15
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:15639]:
```suggestion
```

File: po/ko.po [hunk 3]
```diff
@@ -15632,8 +15636,11 @@ msgid "_Embed.toml_:"
...
 msgid "In one terminal under `src/bare-metal/microcontrollers/examples/`:"
-msgstr "`src/bare-metal/microcontrollers/examples/` 아래 하나의 터미널에서:"
+msgstr ""
+"`src/bare-metal/aps/examples`에서 `make qemu`를 사용하여 QEMU에서 예를 실행합"
+"니다."
```
Comment (jiyongp) [inline po/ko.po:15643]:
```suggestion
msgstr "`src/bare-metal/microcontrollers/example/` 에 터미널을 열고:"
```

File: po/ko.po [hunk 4]
```diff
@@ -15750,7 +15757,7 @@ msgstr ""
 msgid ""
 "We will read the direction from an I2C compass, and log the readings to a "
 "serial port."
-msgstr ""
+msgstr "I2C 나침반에서 방향을 읽고 판독값을 직렬 포트에 기록하세요."
```
Comment (jiyongp) [inline po/ko.po:15760]:
```suggestion
msgstr "I2C 나침반에서 방향을 읽고, 읽은 값을 직렬 포트에 기록하세요."
```

File: po/ko.po [hunk 5]
```diff
@@ -15825,49 +15848,53 @@ msgstr "_hello_rust/src/main.rs_:"
 #: src/exercises/bare-metal/compass.md:47
 #: src/exercises/bare-metal/solutions-morning.md:32
 msgid "// Configure serial port.\n"
-msgstr ""
+msgstr "// 직렬 포트를 구성합니다.\n"
```
Comment (jiyongp) [inline po/ko.po:15851]:
```suggestion
msgstr "// 직렬 포트를 설정하세요.\n"
```

File: po/ko.po [hunk 6]
```diff
@@ -15876,34 +15903,35 @@ msgstr "Bare Metal Rust 오전 연습"
...
-msgstr ""
+msgstr "// 나침반 데이터를 읽고 직렬 포트에 기록합니다.\n"
 
 #: src/exercises/bare-metal/solutions-morning.md:67
 msgid "\"{},{},{}\\t{},{},{}\""
-msgstr ""
+msgstr "\"{},{},{}\\t{},{},{}\""
 
 #: src/exercises/bare-metal/solutions-morning.md:105
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:15929]:
```suggestion
```

File: po/ko.po [hunk 7]
```diff
@@ -16036,18 +16135,25 @@ msgid ""
 "on zeroes. The compiler assumes that the loader will take care of zeroing "
 "them."
 msgstr ""
+"BSS(이전 이유로 인한 블록 시작 기호)는 0으로 초기화된 정적으로 할당된 변수를 "
+"포함하는 객체 파일의 일부입니다. 0에 공간을 낭비하지 않도록 이미지에서 생략됩"
+"니다. 컴파일러는 로더가 0으로 설정하는 작업을 처리한다고 가정합니다."
```
Comment (jiyongp) [inline po/ko.po:16140]:
```suggestion
"BSS(역사적인 이유로, 블록 시작 기호 block starting symbol 이라고 불림)는 오브젝트 파일에서 "
"0으로 초기화된 정적으로 할당된 변수들을 담고 있습니다. BSS는 실제로 이미지에 포함되지는 않습니다. "
"어차피 0으로 초기화 될 것이기 때문입니다. 컴파일러는 로더 프로그램이 BSS에 속하는 변수들을 0으로 초기화 할 것으로"
"기대하고 오브젝트 파일을 만듭니다."
```

File: po/ko.po [hunk 8]
```diff
@@ -16198,12 +16322,13 @@ msgstr ""
...
 #: src/bare-metal/aps/inline-assembly.md:60
+#, fuzzy
 msgid ""
 "Run the example in QEMU with `make qemu_psci` under `src/bare-metal/aps/"
 "examples`."
 msgstr ""
-"`src/bare-metal/aps/examples`에서 `make qemu_psci`를 사용하여 QEMU에서 예시"
-"를 실행합니다."
+"`src/bare-metal/aps/examples`에서 `make qemu`를 사용하여 QEMU에서 예를 실행합"
+"니다."
```
Comment (jiyongp) [inline po/ko.po:16331]:
```suggestion
"`src/bare-metal/aps/examples`에서 `make qemu_psci`를 입력하면 예제 코드가 QEMU에서 수행됩니다."
```

File: po/ko.po [hunk 9]
```diff
@@ -16218,12 +16343,13 @@ msgid "Never hold a reference."
 msgstr "참조를 유지하지 마세요."
 
 #: src/bare-metal/aps/mmio.md:5
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:16346]:
```suggestion
```

File: po/ko.po [hunk 10]
```diff
@@ -17065,12 +17265,13 @@ msgstr ""
 "와 같은 기능이 필요할 경우 어떻게 해야 할까요? "
 
 #: src/bare-metal/useful-crates/spin.md:7
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:17268]:
```suggestion
```

File: po/ko.po [hunk 11]
```diff
@@ -17065,12 +17265,13 @@ msgstr ""
...
 msgid ""
 "The [`spin`](https://crates.io/crates/spin) crate provides spinlock-based "
 "equivalents of many of these primitives."
 msgstr ""
-"[`spin`](https://crates.io/crates/spin) 크레이트는 이러한 동기화 프리미티브들"
-"을 스핀록으로 구현하고 있습니다."
+"[`bitflags`](https://crates.io/crates/bitflags) 크레이트는 비트플래그를 사용"
+"하는 데 유용합니다."
```
Comment (jiyongp) [inline po/ko.po:17274]:
```suggestion
"[`spin`](https://crates.io/crates/spin) 크레이트는 이러한 동기화 프리미티브들"
"을 스핀록으로 구현하고 있습니다."
```

File: po/ko.po [hunk 12]
```diff
@@ -17370,35 +17605,45 @@ msgid ""
...
-msgstr ""
+msgstr "\"linux\""
 
 #: src/exercises/bare-metal/rtc.md:437 src/exercises/bare-metal/rtc.md:439
 msgid "\"CROSS_COMPILE\""
-msgstr ""
+msgstr "\"CROSS_COMPILE\""
 
 #: src/exercises/bare-metal/rtc.md:437
 #, fuzzy
```
Comment (jiyongp) [inline po/ko.po:17635]:
```suggestion
```

File: po/ko.po [hunk 13]
```diff
@@ -17370,35 +17605,45 @@ msgid ""
...
 msgid "\"aarch64-linux-gnu\""
-msgstr "aarch64-paging"
+msgstr "\"aarch64-none-elf\""
```
Comment (jiyongp) [inline po/ko.po:17637]:
```suggestion
msgstr "\"aarch64-linux-gnu\""
```

File: po/ko.po [hunk 14]
```diff
@@ -17370,35 +17605,45 @@ msgid ""
...
 
 #: src/exercises/bare-metal/rtc.md:439
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:17640]:
```suggestion
```

File: po/ko.po [hunk 15]
```diff
@@ -17578,68 +17823,225 @@ msgid ""
...
-msgid "_exceptions.S_ (you shouldn't need to change this):"
-msgstr ""
-
-#: src/exercises/bare-metal/rtc.md:602
-msgid ""
 "```armasm\n"
 "/*\n"
 " * Copyright 2023 Google LLC\n"
 " *\n"
-" * Licensed under the Apache License, Version 2.0 (the \"License\");\n"
```
Comment (jiyongp) [inline po/ko.po:17592]: Don't translate copyright license

File: po/ko.po [hunk 16]
```diff
@@ -17578,68 +17823,225 @@ msgid ""
...
-" * you may not use this file except in compliance with the License.\n"
-" * You may obtain a copy of the License at\n"
+" * Apache 라이선스 버전 2.0(\"라이선스\")에 따라 라이선스가 부여되었습니다.\n"
+" * 라이선스를 준수하지 않으면 이 파일을 사용할 수 없습니다.\n"
+" * 다음 주소에서 라이선스 사본을 가져올 수 있습니다.\n"
 " *\n"
 " *     https://www.apache.org/licenses/LICENSE-2.0\n"
 " *\n"
-" * Unless required by applicable law or agreed to in writing, software\n"
```
Comment (jiyongp) [inline po/ko.po:17598]: Same here

File: po/ko.po [hunk 17]
```diff
@@ -17829,10 +18413,56 @@ msgid ""
 "\t.fill\t\t255, 8, 0x0\t\t\t// 255 GiB of remaining VA space\n"
 "```"
 msgstr ""
+"```armasm\n"
+"/*\n"
```
Comment (jiyongp) [inline po/ko.po:18417]: Same here

File: po/ko.po [hunk 18]
```diff
@@ -17944,18 +18574,125 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
+"```ld\n"
+"/*\n"
+" * Copyright 2023 Google LLC\n"
+" *\n"
+" * Apache 라이선스 버전 2.0(\"라이선스\")에 따라 라이선스가 부여되었습니다.\n"
```
Comment (jiyongp) [inline po/ko.po:18581]: Same here

File: po/ko.po (hunk 15 above)
Comment (henrif75) [reply po/ko.po:17592]: I think we should keep the code without translation.

**Review by jiyongp — APPROVED** [review]

**Comment by henrif75** [comment]
@jiyongp That was quick, thank you!

File: po/ko.po (hunk 15 above)
Comment (mgeisler) [reply po/ko.po:17592]: We can remove the code from the translation files, see #1853.

**Pull request merged**